include/z64player.h

```diff
@@ -360,8 +360,8 @@ typedef struct Player {
     /* 0x08A2 */ s16        unk_8A2;
     /* 0x08A4 */ f32        unk_8A4;
     /* 0x08A8 */ f32        unk_8A8;
-    /* 0x08AC */ f32        windSpeed;
-    /* 0x08B0 */ s16        windDirection;
+    /* 0x08AC */ f32        pushedSpeed; // Pushing player, examples include water currents, floor converyors, climbing sloped surfaces
+    /* 0x08B0 */ s16        pushedDirection; // Yaw direction of player being pushed
```

Comment:
If we did decide to use yaw for the names, possible names that could change to remain consistent involves:
`SurfaceType_GetConveyorDirection` -> `SurfaceType_GetConveyorYaw`
`sConveyorDirection` ->: `sConveyorYaw`
(or `DirYaw` respectively).

Thoughts on changing both those names as well?

Reply:
`sConveyorDirection` ->: `sConveyorYaw`/DirYaw yeah

`SurfaceType_GetConveyorDirection` returns a 0-63 value that imo makes more sense to think of as an abstract direction "encoded yaw" than an actual yaw, even if it ends up being proportional to an actual yaw

`*GetSlopeDirection` is fine for me too because it also returns the normal

Reply:
Is it really necessary to call everything that acts on y rot alone yaw?

Reply:
As an alternative, what about something like `XZDirection`?

Reply:
imo "yaw" is better than "rotY" because it's one letter shorter and also conveys being based on the upwards vertical axis, not just the mathematical +y axis

"yaw" is also an angle, like "rotY"

"direction" for me is a vector.